about/package-scope.md

```diff
@@ -30,9 +28,229 @@ open a [pre-submission inquiry using a GitHub Issue](https://github.com/pyOpenSc
+well here  - the link below for it is a presubmission not a review   -->
+* Examples: [Hamilton - currently under review](https://github.com/pyOpenSci/software-submission/issues/74), martin's git package (that is no longer maintained.)
+
+### Citation management and bibliometrics:
+
+Tools that facilitate managing references, such as for writing manuscripts,
+creating CVs or otherwise attributing scientific contributions, or accessing,
+manipulating or otherwise working with bibliometric data. (Example: [Example from rOpenSci - RefManageR](https://github.com/ropensci/software-review/issues/119))
+
+### Data visualization & analysis
+These are packages that enhance a scientists experience visualizing and
+analyzing data.
+
+* Examples: [PyGmt - (also spatial and data munging)](https://github.com/pyOpenSci/software-submission/issues/43),
+
+### Database software bindings
+
+ Bindings and wrappers for database APIs.
+
+ * Example: [Example from rOpenSci - rrlite](https://github.com/ropensci/software-review/issues/6)
+
+
+## Specialty topics
+
+In addition, we have some specialty topics with a broad domain focused scope.
+
+### Geospatial
+
+Packages focused on the retrieval, manipulation, and analysis of spatial data.
+
+* Examples: [PyGmt](https://github.com/pyOpenSci/software-submission/issues/43),
+[Moving Pandas ](https://github.com/pyOpenSci/software-submission/issues/18)
+
+### Pangeo
+
+We have a [partnership with Pangeo](../partners/pangeo). Often times packages submitted as a part of that partnership are also in the geospatial domain.
+
+* Examples: [xclim - under review now](https://github.com/pyOpenSci/software-submission/issues/73)
+
+### Education
+
+Packages to aid with instruction.
+<!--TODO -  Earthpy fit in this category but it also needs to be sunsetted -->
+
+## Package technical scope
+
+To be in technical scope for a pyOpenSci review, your package:
+
+* Should have maintenance workflows documented.
+* Should be structured in a way that someone else could contribute to it.
+* Should declare vendor dependencies using standard approaches rather than including code from other packages within your repository.
+
+### Notes on scope categories
+- pyOpenSci is still developing as a community. If your scientific Python
+package does not fit into one of the categories or if you have any other
+questions, we'd encourage you to open a pre-submission inquiry. We're happy to help.
+- Data visualization packages come in many varieties, ranging from small
+hyper-specific methods for one type of data to general, do-it-all packages
+(e.g. matplotlib). pyOpenSci accepts packages that are somewhere in between the
+two. If you're interested in submitting your data visualization package, please
+open a pre-submission inquiry first.
+
+## Python package technical scope
+
+pyOpenSci may continue to update its technical scope criteria for package
```

Comment:
```suggestion
pyOpenSci may continue to update its criteria for technical scope 
```
nit: suggest rephrasing to avoid noun stack "technical scope criteria"